packages/ui/package.json

```diff
@@ -40,12 +40,14 @@
     "class-variance-authority": "^0.4.0",
     "cmdk": "^0.2.0",
     "cmk": "^0.1.1",
+    "date-fns": "^3.6.0",
```

Comment:
We already have this as part of a peer-dep for tremor. But we need this as a dependancy as react-day-picker needs it.